rebench/subprocess_kill.py

```diff
@@ -0,0 +1,69 @@
+from os         import kill
+from signal     import SIGKILL
+from subprocess import PIPE, Popen
+
+IS_PY3 = None
+
+try:
+    _ = ProcessLookupError
+    IS_PY3 = True
+except NameError:
+    IS_PY3 = False
+
+# Indicate timeout with standard exit code
+E_TIMEOUT = -9
+
+
+def _kill_py2(proc_id, uses_sudo):
+    if uses_sudo:
+        from .denoise import deliver_kill_signal  # pylint: disable=import-outside-toplevel
+
+        deliver_kill_signal(proc_id)
+        return
+
+    try:
+        kill(proc_id, SIGKILL)
+    except IOError:
+        # it's a race condition, so let's simply ignore it
```

Comment:
A race condition? Can you make the comment more specific, I'm not sure where it's coming from here

Reply:
Changed the comment to: 
https://github.com/smarr/ReBench/pull/196/files#diff-bb5624338cfad103d5b72abfbc7bb001cfc18d20edf8ea325c9a7e0f174c3fe1R42-R43

"""
# there's a race condition, the process may have already terminated on its own
# so let's simply ignore it
"""

Does that answer your question?